### PR TITLE
fix issues on frontend

### PR DESCRIPTION
**Pull request by totop716**
- Add swap route <br/> screenshot
- Do not show APY on pools page for ended farms <br/>screenshot
- Fix token details on swap page <br/>screenshot
- Fix typos on landing page 
- Update transactions table on analytics pair details
- Update color of sort columns on farms and dragons syrup <br/>screenshot
- Fix links of breakcrumb on analytics token and pair details.<br/>screenshot
- Add new tokens, syrups and farms

### CODE REVIEW

File: src/components/PoolPositionCard/PoolPositionCard.tsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ const PoolPositionCard: React.FC<{ pair: Pair }> = ({ pair }) => {
       </Box>
 
       {showMore && <PoolPositionCardDetails pair={pair} />}
-      {stakingInfo && apyWithFee && (
+      {stakingInfo && !stakingInfo.ended && apyWithFee && (
```
Comment (FriskyHamTitz) [inline src/components/PoolPositionCard/PoolPositionCard.tsx:111]: The code is unclear here.  What does pools have to do with staking info ending.  I can understand that for farms or syrup but why pools.

Please respond to my comment below

File: src/components/Swap/AdvancedSwapDetails.tsx [hunk 2]
```diff
@@ -70,6 +71,7 @@ export const TradeSummary: React.FC<TradeSummaryProps> = ({
     allowedSlippage,
   );
   const classes = useStyles();
+  const showRoute = Boolean(trade && trade.route.path.length > 2);
```
Comment (FriskyHamTitz) [inline src/components/Swap/AdvancedSwapDetails.tsx:74]: Why not just always show the route?

File: src/components/Swap/SwapRoute.tsx [hunk 3]
```diff
@@ -1,48 +1,45 @@
...
-            <Box>
-              <CurrencyLogo currency={token} size='1.5rem' />
+          <Box key={i} display='flex' alignItems='center' my={0.5}>
+            <Box display='flex' alignItems='center'>
+              <CurrencyLogo currency={token} size='24px' />
               <Typography>{token.symbol}</Typography>
             </Box>
-            {isLastItem ? null : <ChevronRight />}
-          </Fragment>
+            {!isLastItem && <ChevronRight />}
```
Comment (FriskyHamTitz) [inline src/components/Swap/SwapRoute.tsx:37]: This code looks like a hack. Should probably be explained or written in a proper form

File: src/components/TransactionsTable/TransactionsTable.tsx [hunk 4]
```diff
@@ -106,11 +106,24 @@ const headCells = (
...
+  const getTxString = (txn: any) => {
+    return `${txn.type} ${txn.pair.token0.symbol} ${
+      txn.type === 'Swap' ? 'for' : 'and'
+    } ${txn.pair.token1.symbol}`;
+  };
+  const getTxAmount0String = (txn: any) => {
+    const token = txn.type === 'Swap' ? txn.pair.token1 : txn.pair.token0;
+    return `${formatNumber(txn.amount0)} ${token.symbol}`;
+  };
+  const getTxAmount1String = (txn: any) => {
```
Comment (FriskyHamTitz) [inline src/components/TransactionsTable/TransactionsTable.tsx:123]: There's no need for 2 methods for this just pass the amount in.

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: src/components/PoolPositionCard/PoolPositionCard.tsx (hunk 1 above)
Comment (totop716) [reply src/components/PoolPositionCard/PoolPositionCard.tsx:111]: We are showing this part if the farm is existing with the same pair of the pool. The fix is not to show this section for the ended farms.
screenshot

File: src/components/Swap/AdvancedSwapDetails.tsx (hunk 2 above)
Comment (totop716) [reply src/components/Swap/AdvancedSwapDetails.tsx:74]: It shows the route when there is more than 2 items in the path in our current interface.

File: src/components/PoolPositionCard/PoolPositionCard.tsx (hunk 1 above)
Comment (totop716) [reply src/components/PoolPositionCard/PoolPositionCard.tsx:111]: screenshot
This is to show APY when the farm with same pair of the pool is existing. This update is not to show the section for the ended farm.

File: src/components/Swap/SwapRoute.tsx (hunk 3 above)
Comment (totop716) [reply src/components/Swap/SwapRoute.tsx]: Put a comment for this part and created a commit

File: src/components/TransactionsTable/TransactionsTable.tsx (hunk 4 above)
Comment (totop716) [reply src/components/TransactionsTable/TransactionsTable.tsx:123]: We are showing different token for 2 amount columns in transactions table on analytics pair details. That's why I created 2 functions for each amount column

File: src/components/SwapTokenDetails/SwapTokenDetails.tsx [hunk 5]
```diff
@@ -67,9 +64,20 @@ const SwapTokenDetails: React.FC<{
         latestBlock,
       );
       setPriceData(tokenPriceData);
+
+      const [newPrice, oneDayPrice] = await getEthPrice();
+      const tokenInfo = await getTokenInfo(newPrice, oneDayPrice, tokenAddress);
+      if (tokenInfo) {
+        setTokenData(tokenInfo[0]);
```
Comment (FriskyHamTitz) [inline src/components/SwapTokenDetails/SwapTokenDetails.tsx:71]: Try not to use indexers multiple times e.g 
let token0 = token info[0]

File: src/components/TransactionsTable/TransactionsTable.tsx (hunk 4 above)
Comment (FriskyHamTitz) [reply src/components/TransactionsTable/TransactionsTable.tsx:123]: You don't gain much by calling the method 0 or 1 just pass the amount in instead.

const getTxAmountString = (txn: any,  amount: Number) => {
    const token = txn.type === 'Swap' ? txn.pair.token1 : txn.pair.token0;
    return `${formatNumber(amount)} ${token.symbol}`;
  };

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: src/components/SwapTokenDetails/SwapTokenDetails.tsx (hunk 5 above)
Comment (totop716) [reply src/components/SwapTokenDetails/SwapTokenDetails.tsx]: Pushed a commit with update

File: src/components/TransactionsTable/TransactionsTable.tsx (hunk 4 above)
Comment (totop716) [reply src/components/TransactionsTable/TransactionsTable.tsx]: Moved transaction messages to `en.json` for localization and pushed updates

File: src/components/TransactionsTable/TransactionsTable.tsx [hunk 6]
```diff
@@ -105,12 +107,32 @@ const headCells = (
...
+  const [txFilter, setTxFilter] = useState(-1);
+  const txHeadCells = headCells(txFilter, setTxFilter);
   const classes = useStyles();
   const { chainId } = useActiveWeb3React();
   const { palette, breakpoints } = useTheme();
   const isMobile = useMediaQuery(breakpoints.down('xs'));
+  const { t } = useTranslation();
+  const getTxString = (txn: any) => {
+    if (txn.type === TxnType.SWAP) {
+      return t('txnSwapMessage', {
```
Comment (FriskyHamTitz) [inline src/components/TransactionsTable/TransactionsTable.tsx:119]: In the future you can put that object into a variable instead of duplicating it

**Review by FriskyHamTitz — APPROVED** [review]

File: src/components/TransactionsTable/TransactionsTable.tsx (hunk 6 above)
Comment (totop716) [reply src/components/TransactionsTable/TransactionsTable.tsx:119]: Can you tell me which object you meant to put into a variable?

Comment (totop716) [reply src/components/TransactionsTable/TransactionsTable.tsx]: Pushed a commit with update

**Pull request merged**